lib/Model/Importer/Binary/PECOFFImporter.cpp

```diff
@@ -35,6 +35,12 @@ class PECOFFImporter : public BinaryImporterHelper {
   Error parseSectionsHeaders();
   /// Parse static symbols from the file.
   void parseSymbols();
+
+  /// Parse dynamic symbols from the file.
+  void parseImportedSymbols();
+  void recordImportedFunction(
+    iterator_range<llvm::object::imported_symbol_iterator> Range,
+    uint32_t ImportAddressTableAddr);
```

Comment:
* Does this pass `revng-check-conventions`?
* `s/recordImportedFunction/recordImportedFunctions`?

Reply:
>Does this pass revng-check-conventions?

It complains, but I thought the --force-format resolves everything. Done.

>s/recordImportedFunction/recordImportedFunctions?

Sure. tnx.